lib/rouge/demos/xojo

```diff
@@ -1,7 +1,8 @@
 Dim f As FolderItem
 f = GetOpenFolderItem(FileTypes1.jpeg) // defined in the File Type Set editor
+rem - we should check for nil!
```

Comment:
What's the significance of the hyphen?

Reply:
No significance. That line should be considered a "rem comment", so any text is allowed.